### ISSUE TITLE
Logistic Regression: One vs Rest Classifier

### ISSUE THREAD

**Issue by Technocolabs100** [Technocolabs100/Stack-Overflow-Tag-Predictions#19]
We have very high dimensional data and we need to build many models in a binary representation. To tackle this, we have to take the help of Logistic Regression with One vs Rest classifier. The classifier takes each of the labels and trains 5500 logistic regression models. Training a Logistic Regression model is very cheap and easy when compared to other models like Support Vector Machines (SVM), Random Forest, etc..and it performs really well on high dimensional data.

**Comment by RidhimaKohli** [comment]
Can I work on this issue ?

**Comment by Technocolabs100** [comment]
Hello Ridhima,
I will be assign this issue to you!
Thanks

**Comment by RidhimaKohli** [comment]
I have a few doubts :
1) Can I use sklearn ?
2) Do I need to use that kaggle dataset mentioned in readme ?
3) Will this issue be also counted for GS Soc ?

**Comment by Technocolabs100** [comment]
Yes, You need to use sklearn for model building part and use the data sets from kaggle link is already mentioned in the readme file.

Definitely this issue is to be count in GSSoC.

**Comment by A-kriti** [comment]
Hello @Technocolabs100 , I would like to contribute to this issue as a GSSOC'21 participant. So could you please assign me this?

**Comment by RidhimaKohli** [comment]
Okay thank you

**Comment by Technocolabs100** [comment]
@RidhimaKohli Hope you are doing well!
We need to know the working status of this issue as soon as possible.
